src/card.css

```diff
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+  flex-wrap: wrap;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  margin: 40px 16px 0;
+  line-height: 18px;
+  color: var(--color-dark);
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--color-gray);
+  margin: 4px 16px 0;
+  line-height: 14px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 17px 16px 0;
+  width: 100%;
+}
+
+.card__review-text {
+  font-size: 10px;
+  font-weight: 400;
+  text-align: right;
+  line-height: 14px;
+  color: var(--color-dark);
+}
+
+.card__price {
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--color-gray);
+  margin: 24px 86px 0 16px;
+  line-height: 18px;
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  margin-top: 24px;
+  line-height: 18px;
+  color: var(--color-dark);
+}
+
+.card__link {
+  display: flex;
+  text-decoration: none;
+  justify-content: center;
+  text-align: center;
+  width: 100%;
+  padding: 11px 0;
+  background-color: rgba(0, 172, 220, 1);

```

Comment:
why don't you move this color to variables like other colors that you use not once?